testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/security/AddAuthorizationModuleOnlineTest.java

```diff
@@ -44,6 +47,15 @@ public class AddAuthorizationModuleOnlineTest {
     private static final Address TEST_AUTHORIZATION_MODULE_ADDRESS_2 = TEST_AUTHZ_CLASSIC_ADDRESS
             .and("policy-module", TEST_AUTHORIZATION_MODULE_NAME_2);
 
+    @BeforeClass
+    public static void checkServerVersionIsSupported() throws Exception {
+        // check version is supported
+        ServerVersion serverVersion
+                = ManagementClient.online(OnlineOptions.standalone().localDefault().build()).version();
+        Assume.assumeFalse("Legacy security was removed in WildFly 25.",
+                serverVersion.greaterThanOrEqualTo(ServerVersion.VERSION_18_0_0));
+    }
```

Comment:
Just a proposal - wouldn't be better to create some common function for this and use it in all relevant places instead of having the very same code on multiple places? :angel:

Reply:
I was thinking about it but in the end, I followed the same convention as it is in elytron commands, there is a similar check in every elytron command.